### PR TITLE
#179716166 remove initial vas constant

### PR DESCRIPTION
**Pull request by tobieeniafe**
#### What does this PR do?
Remove `INITIAL_VAS` constant

#### Description of Task to be completed?
Remove `INITIAL_VAS` constant
Remove `activated` and `activationDate` from userInfo when viewed by others

#### How should this be manually tested?
`npm run test`

#### What are the relevant pivotal tracker stories?
#179716166

### CODE REVIEW

File: server/helpers/projectedSchemaInfo.js [hunk 1]
```diff
@@ -79,6 +79,8 @@ export const NON_PROJECTED_USER_INFO = (infoType = null, userRole = null) => {
           [`${dataPrefix}phone`]: 0,
           [`${dataPrefix}phone2`]: 0,
           [`${dataPrefix}address`]: 0,
+          [`${dataPrefix}activated`]: 0,
```
Comment (andela-hpopoola) [inline server/helpers/projectedSchemaInfo.js:82]: Can we add tests to confirm this please?

File: server/helpers/projectedSchemaInfo.js [hunk 2]
```diff
@@ -103,7 +105,7 @@ export const NON_PROJECTED_USER_INFO = (infoType = null, userRole = null) => {
 };
 
 export const getExcludedTransactionInfo = (userRole) =>
-  userRole === USER_ROLE.USER ? { remittance: 0 } : {};
+  userRole === USER_ROLE.USER ? { remittance: 0, activated: 0, activationDate: 0 } : {};
```
Comment (andela-hpopoola) [inline server/helpers/projectedSchemaInfo.js:108]: `activated` and `activationDate` are not part of the transaction model, not sure why you added the code to remove them here when they are not even present 🤷🏽‍♂️

**Review by andela-hpopoola — APPROVED** [review]
Thank you for working on this. When we make model changes, it is best practice to always accompany such changes with tests.

File: server/helpers/projectedSchemaInfo.js (hunk 2 above)
Comment (tobieeniafe) [reply server/helpers/projectedSchemaInfo.js:108]: My mistake 🤦🏾‍♂️

**Pull request merged**